data/problems/en/012_run_length_decoding.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Decode a run-length encoded list
+number: "12"
+difficulty: intermediate
+tags: [ "list" ]
+---
+
+# Solution
+
+```ocaml
+# let decode list =
```

Comment:
When this was all one file it probably inherited the types from previous questions and solutions, now they are separate we will have to redefine them